debezium-core/src/test/java/io/debezium/data/VerifyRecord.java

```diff
@@ -250,6 +250,77 @@ public static void isValidTombstone(SourceRecord record, String pkField, int pk)
         isValidTombstone(record);
     }
 
+    /**
```

Comment:
I've shuffled some code around from AbstractConnectorTest, if this isn't the right approach here happy to back it out.

Reply:
Seems good to me :)